metadata/org.eclipse.jetty/jetty-client/12.0.0.beta0/resource-config.json

```diff
@@ -0,0 +1,14 @@
+{
+  "resources":{
+  "includes":[{
+    "condition":{"typeReachable":"org.eclipse.jetty.client.http.HttpReceiverOverHTTP"},
+    "pattern":"\\QMETA-INF/services/org.eclipse.jetty.http.HttpFieldPreEncoder\\E"
```

Comment:
This should be covered by the `ServiceLoader` integration GraalVM has. Can be removed.